src/libs/Navigation/AppNavigator/BaseDrawerNavigator.js

```diff
@@ -49,6 +49,23 @@ class BaseDrawerNavigator extends Component {
         };
     }
 
+    componentDidUpdate(prevProps) {
+        // You don't have to do this check first, but it can help prevent an unneeded render
+        if (prevProps.isSmallScreenWidth === this.props.isSmallScreenWidth) {
+            return;
+        }
+        this.updateNavigationState(this.props.isSmallScreenWidth);
```

Comment:
do we really need this function? it's just setting a state.

can we directly set the state instead?

Reply:
Yes, we can set state directly but it was giving error at lint checks of Do not use setState in componentDidUpdate. To resolve it i created this function.

Reply:
makes sense. we should disable the lint check here then

 `// eslint-disable-next-line [rule name]` ([SO](https://stackoverflow.com/questions/27732209/turning-off-eslint-rule-for-a-specific-line))

Reply:
Thanks, For the help. Just pushed the latest code with fix.